### ISSUE TITLE
NVDA's Read Title command does not read the title of the active or open Web page in Seamonkey 2.1

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1578]
**Reported by dgoldfield on 2011-06-17 13:00**
When I press NVDA modifier key plus T to obtain the title, NVDA says "Seamonkey" and not the title of the open or active Web page.  This occurs with the new Seamonkey version 2.1 but did not occur with the 2.0.x versions.

**Comment by leonardder** [comment]
This issue seems abandoned and outdated, as seamonkey has received several updates since 2011

**Comment by jcsteh** [comment]
Closing. If this still persists in newer versions, please comment here and we can reopen and adjust the title.
